stories/license.stories.jsx

```diff
@@ -1,20 +1,33 @@
 import React from 'react';
 
 import { storiesOf } from '@storybook/react';
+import { withKnobs, text, number } from '@storybook/addon-knobs';
 import { withInfo } from "@storybook/addon-info";
 import { action } from '@storybook/addon-actions';
 
 import License from '../app/js/components/License';
 
-storiesOf('License', module)
+const stories = storiesOf('License', module)
+
+stories.addDecorator(withKnobs);
```

Comment:
What do knobs do?

Reply:
It allows to make change to the props directly in storybook